ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -392,73 +324,13 @@ const PanelMain = fp.flow(
         analysisConfig.diskConfig.detachable) ||
         (analysisConfig.computeType === ComputeType.Dataproc &&
           !analysisConfig.diskConfig.detachable));
-    // Casting to RuntimeStatus here because it can't easily be done at the destructuring level
-    // where we get 'status' from
     const runtimeCanBeUpdated =
       environmentChanged &&
       (
         [RuntimeStatus.RUNNING, RuntimeStatus.STOPPED] as Array<RuntimeStatus>
-      ).includes(status as RuntimeStatus) &&
+      ).includes(runtimeStatus) &&
       runtimeCanBeCreated;
 
-    const renderCreateOrUpdateButton = (
```

Comment:
I know I just wrote this function, but I changed my mind :)

These are all simple `<Button>`s now